src/main/java/com/kenya/jug/arena/model/Game.java

```diff
@@ -0,0 +1,22 @@
+package com.kenya.jug.arena.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Game {
+    @Embedded
+    private EntityMetadata entityMetadata;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
The specification recommends using SQLite database persistence for easy deployment. Refactor the code to store data in SQLite instead of PostgresSQL